packages/admin/test/rule.js

```diff
@@ -1,7 +1,7 @@
 const expect = require('chai').expect;
 const semver = require('semver');
 
-function checkCondition(objMessages, oldVersion, newVersion) {
+function checkCondition(objMessages, oldVersion, newVersion, instances) {
```

Comment:
(nit and more for future): I think in general we should aim for using options objects if we have a lot of arguments passed to the function and especially if some are optional like here.